src/lib.rs

```diff
@@ -320,6 +327,36 @@ pub async fn compare_attestations(
     let mut remote_attestations = remote_attestations.clone();
     remote_attestations.sort_by(|a, b| a.stake_weight.partial_cmp(&b.stake_weight).unwrap());
 
+    if !remote_attestations.is_empty() {
+        let subgraph_gauge = ACTIVE_SUBGRAPHS.with_label_values(&[&ipfs_hash]);
+
+        let all_senders: Vec<String> = remote_attestations
+            .clone()
+            .into_iter()
+            .flat_map(|attestation| attestation.senders)
+            .collect();
+
+        let unique_senders: HashSet<String> = HashSet::from_iter(all_senders.into_iter());
+
+        // The value is the total number of unique senders that are attesting for that subgraph
+        subgraph_gauge.set(unique_senders.len().try_into().unwrap())
+    }
+
+    if !remote_attestations.is_empty() {
```

Comment:
and this block is repeated from the block on L330?

Reply:
ah darn, git did me dirty here, thanks!!